Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -18,6 +18,11 @@ private var structureCache = Cache({ file -> Structure? in
     }
     return nil
 })
+
+private var structureDictionaryCache = Cache({ file in
+    return structureCache.get(file).map { SourceKittenDictionary($0.dictionary) }
```

Comment:
Nit:

```suggestion
    return structureCache.get(file).map(SourceKittenDictionary.init)
```

Reply:
Not really, since it uses `$0.dictionary`, not the `$0`

Reply:
Oops, missed that.